host/installer/installer.nsi

```diff
@@ -0,0 +1,196 @@
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+Place, Suite 330, Boston, MA 02111-1307 USA
+*/
+
+
+;Include  
+!include "MUI2.nsh"
+!include "FileFunc.nsh"
+!include "LogicLib.nsh"
+!include "Sections.nsh"
+
+;Settings
+Name "Looking Glass Host"
+OutFile "..\build\looking-glass-host-setup.exe" 
+Unicode true
+RequestExecutionLevel admin
+ShowInstDetails "show"
+ShowUninstDetails "show"
+InstallDir "$PROGRAMFILES64\Looking Glass"
+
+!define MUI_ICON "..\..\resources\icon.ico"
+!define MUI_UNICON "..\..\resources\icon.ico"
+!define MUI_LICENSEPAGE_BUTTON "Agree"
+!define /file VERSION "..\..\VERSION"
+
+;Install and uninstall pages
+!insertmacro MUI_PAGE_LICENSE "..\..\LICENSE"
+!insertmacro MUI_PAGE_DIRECTORY
+!insertmacro MUI_PAGE_COMPONENTS
+!insertmacro MUI_PAGE_INSTFILES
+!insertmacro MUI_UNPAGE_CONFIRM
+!insertmacro MUI_UNPAGE_INSTFILES
+!insertmacro MUI_LANGUAGE "English"
+
+
+Function ShowHelpMessage
+	!define line1 "Command line options:$\r$\n$\r$\n"
+	!define line2 "/S - silent install (must be uppercase)$\r$\n"
+	!define line3 "/D=path\to\install\folder - Change install directory$\r$\n"
+	!define line4 "   (Must be uppercase, the last option given and no quotes)$\r$\n$\r$\n"
+	!define line5 "/startmenu - create start menu shortcut$\r$\n"
+	!define line6 "/desktop - create desktop shortcut$\r$\n"
+    !define line7 "/noservice - do not create a service to auto start and elevate the host"
+	MessageBox MB_OK "${line1}${line2}${line3}${line4}${line5}${line6}${line7}"
+	Abort
+FunctionEnd
+
+Function .onInit
+
+    var /GLOBAL cmdLineParams
+    Push $R0
+    ${GetParameters} $cmdLineParams
+    ClearErrors
+	
+    ${GetOptions} $cmdLineParams '/?' $R0
+    IfErrors +2 0
+    Call ShowHelpMessage
+	
+	${GetOptions} $cmdLineParams '/H' $R0
+    IfErrors +2 0
+    Call ShowHelpMessage
+	
+    Pop $R0
+
+
+    Var /GLOBAL option_startMenu
+    Var /GLOBAL option_desktop
+    Var /GlOBAL option_noservice
+    StrCpy $option_startMenu     0
+	StrCpy $option_desktop       0
+    StrCpy $option_noservice     0
+
+    Push $R0
+		
+	${GetOptions} $cmdLineParams '/startmenu' $R0
+    IfErrors +2 0
+	StrCpy $option_startMenu 1
+
+    ${GetOptions} $cmdLineParams '/desktop' $R0
+    IfErrors +2 0
+    StrCpy $option_desktop 1
+    
+    ${GetOptions} $cmdLineParams '/noservice' $R0
+    IfErrors +2 0
+    StrCpy $option_noservice 1
+    
+    Pop $R0
+
+FunctionEnd
+
+;Install 
+Section "-Install" Section1
+
+  SetOutPath $INSTDIR
+  File ..\build\looking-glass-host.exe
+  File /oname=LICENSE.txt ..\..\LICENSE
+  WriteUninstaller $INSTDIR\uninstaller.exe
+
+  ${GetSize} "$INSTDIR" "/S=0K" $0 $1 $2
+  IntFmt $0 "0x%08X" $0
+  WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Looking-Glass" \
```

Comment:
Please change from `Looking-Glass`  to `Looking Glass`, for this and all the following entries.